ensime_shared/ensime.py

```diff
@@ -744,15 +748,16 @@ def clear_breaks(self, args, range=None):
 
     def debug_start(self, args, range=None):
         self.log("debug_start: in")
-        if len(args) > 0:
+        if len(args) > 1:
```

Comment:
Can we make the args optional? In most cases you want to debug localhost on port 5005 I think